btr-web/btr-main-app/stores/omit-individual.ts

```diff
@@ -58,8 +58,18 @@ export const useOmitIndividual = defineStore('bcros/omitIndividual', () => {
   }
 
   /** Load all requests **/
-  async function loadAllRequests () {
-    const url = constructBtrApiURL() + `/requests/${uuid}`
+  async function loadAllRequests (sort?: string, filter?: BtrBodsRequestQueryI, order?: string) {
+    let url = constructBtrApiURL() + '/requests' +
+      `${sort ? '?sort=' + encodeURIComponent(sort) : ''}` +
+      `${sort && order ? '&' : ''}${order && !sort ? '?' : ''}` +
+      `${order ? 'order=' + encodeURIComponent(order) : ''}`
+
+    for (const key in filter) {
+      if (filter[key]) {
+        url += `${url.includes('?') ? '&' : '?'}${key}=${filter[key]}`
+      }
+    }
```

Comment:
You can set all of this as params in the fetch below instead of adding it in the url which simplifies this and removes the need for checking to add '?' vs '&'

ie.
```
params = { ... }
const { data, error } = await useFetchBcros<[BtrBodsRequestGetI]>(url, { method, query: params })
```

Reply:
thats much better changed, thanks